### DISCUSSION TITLE
More customization over RECAP alerts

### DISCUSSION THREAD

**Discussion by ronerofe** [freelawproject/foresight#44] [Ideas]
I use PACER alerts alot as they are very useful, however, I wish sometimes I can more tailor over how and when these alerts are sent to my email. For example, PACER right now does not discriminate between each case filing. Some cases (especially class actions or those filed by DOJ/AGs), can have a litany of Attorney appearance or Pro Hac Vice filings that are of little value to most people. Since CourtListener sees these filings as the same weight if there is a filing of a Court order or opinion, it treats them the same way. It would be nice for a way to filter out the "low-value" filings. I'm thinking this could be based on some word filters in the description or something else. 

Another improvement would be a way to customize when alerts are sent. Right now, it seems like alerts just come in batches of when data is received. Sometimes cases have a lot of filings in one day, and it is a bit overwhelming to get a bunch of emails on that one case, such that it might make more sense to have an option to tell CourtListener to only give alerts once each day or every 12 hours or equivalent.

**Comment by mlissner** [comment]
I appreciate you finding Foresight and being our first user of it! 

We have an issue about adding granularity to alerts here: 

https://github.com/freelawproject/foresight/issues/45

We don't have one about slowing them down. It's an interesting idea. I've resisted anything like either of these because I like that alerts are one-click-and-done, but I think they make sense.
